__sdk__.js

```diff
@@ -47,6 +47,9 @@ const SMART_CARDS = [
 ];
 
 module.exports = {
+    'actions': {
+        entry:          './src/actions/interface',
```

Comment:
Can we have this match the existing pattern with the interface folder?
```suggestion
        entry:          './src/interface/actions',
```